src/appleseed.studio/utility/settingskeys.h

```diff
@@ -42,6 +42,7 @@
 #define SETTINGS_FILE_DIALOG_PROJECTS               "ui.file_dialog.projects"
 #define SETTINGS_FILE_DIALOG_OIIO_TEXTURES          "ui.file_dialog.oiio_textures"
 #define SETTINGS_FILE_DIALOG_PYTHON_SCRIPTS         "ui.file.dialog.python_editor"
```

Comment:
maybe typo?
"ui.file_dialog.python_editor"

Reply:
'.' between file and dialog is typo.. I thought just because other's are "file_dialog"
minor thing.

Reply:
Oh you're right, well spotted! Can you fix it please?